pandas/core/strings.py

```diff
@@ -1022,6 +1052,64 @@ def split(self, pat=None, n=-1, return_type='series'):
         result = str_split(self.series, pat, n=n, return_type=return_type)
         return self._wrap_result(result)
 
+    _shared_docs['str_partition'] = ("""
+    Split the string at the %(side)s occurrence of sep, and return a 3-tuple containing the part
```

Comment:
Other thing: here you explain the '3-tuple' as output, but actually the three columns (expand=True) is the default